lib/Paginator.js

```diff
@@ -140,21 +141,44 @@ Paginator.prototype = {
     },
 
     /**
-     * Paginate ajax page
+     * Return next page index
+     * @returns {Promise}
+     * @private
+     */
+    _getRealNextPageIndex: function () {
+        return this._env.evaluateJs(this._scope, /* @covignore */ function (selector) {
+            var nodes = Sizzle(selector);
+            return Array.prototype.map.call(nodes, function (node) {
```

Comment:
sizzle returns an array, not DOM nodes collection, so you can call `map` directly on it, without `call`